config/config_test.go

```diff
@@ -42,75 +42,75 @@ func TestDefaultConfig(t *testing.T) {
 
 	assert.NoError(t, conf.BasicCheck())
 	assert.Empty(t, conf.Network.ListenAddrStrings)
-	assert.Equal(t, conf.Network.NetworkName, "")
-	assert.Equal(t, conf.Network.DefaultPort, 0)
+	assert.Equal(t, "", conf.Network.NetworkName)
```

Comment:
nit: we could also use `assert.Zero(t, conf.Network.NetworkName)`  to check for zero value